backend/src/packages/auth/auth.service.ts

```diff
@@ -24,14 +29,22 @@ class AuthService {
     if (!user) {
       throw new UserNotFoundError();
     }
+    const accessToken = createToken({ id: user.id });
 
-    return user;
+    return { ...user, accessToken };
```

Comment:
They should be separated
```suggestion
    return { user, token };
```